reposcan/reposcan.py

```diff
@@ -285,13 +285,23 @@ def _parse_input_list(self):
             json_data = self.request.body
 
         data = json.loads(json_data)
+
         for repo_group in data:
             # Entitlement cert is optional
             if "entitlement_cert" in repo_group:
                 cert_name = repo_group["entitlement_cert"]["name"]
-                ca_cert = repo_group["entitlement_cert"]["ca_cert"]
-                cert = repo_group["entitlement_cert"]["cert"]
-                key = repo_group["entitlement_cert"]["key"]
+
+                ca_cert = os.getenv(repo_group["entitlement_cert"]["ca_cert"][1:], "string") if \
```

Comment:
just a detail - instead of "string" default value, please make sure it'll return variable name (if it's not found in environment)

`ca_cert = os.getenv(repo_group["entitlement_cert"]["ca_cert"][1:], repo_group["entitlement_cert"]["ca_cert"])`

same for cert and key